Source/WebCore/css/html.css

```diff
@@ -304,43 +304,45 @@ caption {
 ul, menu, dir {
     display: block;
     list-style-type: disc;
-    margin-block-start: 1__qem;
+    margin-block-start: 1em;
     margin-block-end: 1em;
-    margin-inline-start: 0;
-    margin-inline-end: 0;
     padding-inline-start: 40px;
 }
 
 ol {
     display: block;
     list-style-type: decimal;
-    margin-block-start: 1__qem;
+    counter-reset: list-item;
```

Comment:
This and the other `counter-reset: list-item` rule also need the following part of the spec implemented:

<img width="1436" alt="image" src="https://github.com/WebKit/WebKit/assets/1795135/42f3bc8a-a309-4789-bb72-19f432c3c466">

otherwise you'll regress `imported/w3c/web-platform-tests/css/css-lists/counter-list-item.html`

Reply:
It's not super hard to implement if you want to try, the relevant code is here:

https://searchfox.org/wubkat/rev/e4cb3bdcaddcf79ba58dab47b0245b781c7db975/Source/WebCore/html/HTMLLIElement.cpp#66

https://searchfox.org/wubkat/rev/e4cb3bdcaddcf79ba58dab47b0245b781c7db975/Source/WebCore/html/HTMLOListElement.cpp#70

Though there's a bit of cleanup to do (happy to show you what cleanup), since there's also some pre-existing code that does similar things.

Reply:
I take that back, it's not trivial to implement at all because WebKit doesn't support `reversed()` counters.

Reply:
Removed this for time being and updated commit message as well.